src/phases/scale/MixPhasesRateLimited.yml

```diff
@@ -0,0 +1,79 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  Phase defintions for the MixedWorkloadsGenny, which is a port of the mixed_workloads in the
+  workloads repo. https://github.com/10gen/workloads/blob/master/workloads/mix.js. It runs 4 sets of
+  operations, each with dedicated actors/threads. The 4 operations are insertOne, findOne,
+  updateOne, and deleteOne. Since each type of operation runs in a dedicated thread it enables
+  interesting behavior, such as reads getting faster because of a write regression, or reads being
+  starved by writes. The origin of the test was as a reproduction for BF-2385 in which reads were
+  starved out by writes.
+
+Keywords:
+- scale
+- insertOne
+- insert
+- findOne
+- find
+- updateOne
+- update
+- deleteOne
+- delete
+
+dbname: &dbname mix
+runtime: &runtime 7 minutes
+DocumentCount: &NumDocs 100000
+CollectionCount: &NumColls 1
+
```

Comment:
I wonder if there's a way to keep more of this common between the two workloads. Maybe @brad-devlugt has ideas as he's used the loadconfig and such pretty well before.

Reply:
I'm not sure if there's an easy way to re-use the other files. I think it would be okay to keep separate in case we decide to diverge further or add more rate limited phases in the future.

Reply:
Hi @aggrand , would you be able to review this document to see if you have any ideas about David's thoughts above. The file is a rate limited version of MixedWorkloadsGenny and the above section that has been commented on references the tests loadConfig. Do you have any suggestions/questions?

Reply:
@brad-devlugt @dalyd 
I added in the rate limit to the load config in mixedworkloadsgenny (can view on this PR) and tested locally and with an evergreen patch - https://spruce.mongodb.com/version/6238db6757e85a655ed382af/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC
please let me know if you have an other comments or suggestions.
Thank you!

Reply:
Thanks, looks good to me once we remove MixPhasesRateLimited.yml